<launch>

  <node pkg="mrpt_map" type="object_listener_node" name="object_listener_node" output="screen">

    <param name="frame_id" value="map"/>
    <param name="frequency" value="1"/>
    <!--<param name="ini_file" value="$(find mrpt_map)/tutorial/map.ini"/>-->
    <param name="simplemap_file" value="$(find mrpt_map)/tutorial/constructed_map.simplemap"/>
    <param name="bitmap_file" value="$(find mrpt_map)/tutorial/constructed_map.bmp"/>
  </node>

</launch>
